test/test_nn.py

```diff
@@ -2171,6 +2176,41 @@ def forward(self, x):
         initial_bias_id = id(model.bias)
         initial_model = deepcopy(model)
 
+        # Test allow_resize flag
+        parametrize.register_parametrization(model, "weight", Resize())  # default allow_resize = False
+        self.assertRaises(RuntimeError)
```

Comment:
I don't think this is doing what you want.

You can use
```python
with self.assertRaisesRegex(RuntimeError, "Your error msg"):
    the_code_that_should_fail()
```
To make sure you get the error you expect.